projects/nexus-dashboard/day-2/abhishekgoswami/src/App.css

```
@@ -0,0 +1,112 @@
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body,
+#root {
+  height: 100%;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Stylelint (17.7.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)